### PR TITLE
Jacobbaqleh/About Us and Log In navbar buttons 

### PR DESCRIPTION
**Pull request by JacobBaqleh1**
What changes have been made?

- The About Us and Login button in the navbar links to their page.

What tickets does this PR associate with? (Link to ticket)

- Issue- #12

Challenges (what problems did I face?)

-This PR had an open conflict. I don't see the conflict but if you see any conflict please share a comment.

How has the UI changed? (Screenshot or recording, before or after)

https://github.com/social-plan-it/plan-it-social-web/assets/101436252/3e3fb16c-b9ef-44ca-82ed-069b1cfb997f

Does it work on mobile? (Screenshot for different screen sizes)

https://github.com/social-plan-it/plan-it-social-web/assets/101436252/5ceaebae-befa-4a54-b7b1-0ca0ec4aceb9

Has the database schema changed? If so, is it backward compatible?

Why were these changes made?

- This is the start of building the About Us page.

What tests have been added?

How was it implemented, and why was this implementation chosen?

If a new npm package was added, why was it chosen and why was it necessary?

What alternatives were considered, and why weren't they used?

Documentation of challenges that were faced and how they were fixed

### CODE REVIEW

File: app/components/topNav.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                  alt="Plan It Social Logo!"
+                />
+              </div>
+              <div className="hidden sm:ml-6 sm:block">
+                <div className="flex space-x-4">
+                  {/* <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" --> */}
+                  <Link
+                    to="/"
+                    title="Home"
+                    aria-label="Home"
```
Comment (andrelandgraf) [inline app/components/topNav.tsx:85]: `aria-label` overrides the text content of the element for screen readers. Here we would replace "Home" with "Home", so there is no need for the `aria-label` (same with `title`, I believe).

File: app/components/topNav.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                    className="bg-gray-900 text-white rounded-md px-3 py-2 text-sm font-medium"
+                    aria-current="page"
+                  >
+                    Home
+                  </Link>
+                  <Link
+                    to="/about-us"
+                    title="About Us"
+                    aria-label="About Us"
```
Comment (andrelandgraf) [inline app/components/topNav.tsx:94]:
```suggestion
```

Same here, `title` and `aria-label` are only needed if they add more information. Here, I don't think it's necessary. The screen reader will already read out something like: `Link, About Us` which should be self-explanatory.

File: app/components/topNav.tsx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+                    className="text-gray-300 hover:bg-gray-700 hover:text-white rounded-md px-3 py-2 text-sm font-medium"
+                  >
+                    About
+                  </Link>
+                  <Link
+                    to="/"
+                    title="Login"
+                    aria-label="Login"
```
Comment (andrelandgraf) [inline app/components/topNav.tsx:102]: Same here!

File: app/components/topNav.tsx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+                {/* <!--
+              Dropdown menu, show/hide based on menu state.
+
+              Entering: "transition ease-out duration-100"
+                From: "transform opacity-0 scale-95"
+                To: "transform opacity-100 scale-100"
+              Leaving: "transition ease-in duration-75"
+                From: "transform opacity-100 scale-100"
+                To: "transform opacity-0 scale-95"
+            --> */}
```
Comment (andrelandgraf) [inline app/components/topNav.tsx:160]: Comment can be removed!

**Review by andrelandgraf — COMMENTED** [review]
Left some comments. I believe some are for components introduced in different PRs (my bad) - leaving it just for educational purposes 🤣

**Comment by andrelandgraf** [comment]
Hey @JacobBaqleh1, could you merge the latest `main` branch into this PR, so we see the updated changes? :)

**Review by andrelandgraf — APPROVED** [review]
Great work, LGTM! :)

(we can delete the `topNav.tsx` file later or add it to this PR, whatever works for you!)

File: app/components/topNav.tsx [hunk 5]
```diff
@@ -0,0 +1,245 @@
+import type { LinksFunction } from "@remix-run/node";
```
[the remark this answers is not in the document]
Comment (JacobBaqleh1) [reply app/components/topNav.tsx]: Changed!

File: app/components/topNav.tsx [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+import { Link } from "@remix-run/react";
+import logo_hori from "../imgs/SocialPlan-it-logo-Horizontal.png";
+import logo_sq from "../imgs/SocialPlan-it-logo-Square.png";
+export const links: LinksFunction = () => [
+  {
+    rel: "stylesheet",
+    href: " stylesUrl",
+  },
+];
```
[the remark this answers is not in the document]
Comment (JacobBaqleh1) [reply app/components/topNav.tsx]: Will do.

**Review by andrelandgraf — APPROVED** [review]
LGTM!

**Review by tonydangblog — APPROVED** [review]
Looks good!

**Pull request merged**